.github/workflows/build-apps.yml

```diff
@@ -383,7 +383,7 @@ jobs:
           # see https://github.com/actions/checkout/issues/1471
           git fetch --prune --unshallow --tags
           export TAG="nightly-${VERSION}"
-          export PREVIOUS_TAG=$(git describe --tags --match="nightly-v[0-9]*" --abbrev=0)
```

Comment:
This should fix the changelog by grabbing the prev prev tag.
![image](https://github.com/user-attachments/assets/efaa41fb-ed8c-4315-a2e0-7a038a4c134d)

Reply:
Shoot the current line above doesn't work we have too many tag patterns

Reply:
Fixed now

Reply:
this will be in https://github.com/KittyCAD/modeling-app/pull/5550 actually